src/module_ncio.f90

```diff
@@ -198,6 +255,7 @@ end function has_var
   !! @param varname 
```

Comment:
You can fill in many of these parameters.

@param varname the variable name
@param attname the attribute name

etc.